corretto8/plan.sh

```diff
@@ -7,7 +7,7 @@ pkg_description=('Corretto is a build of the Open Java Development Kit (OpenJDK)
 pkg_license=("GPL-2.0-only")
 pkg_upstream_url=https://aws.amazon.com/corretto/
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source="https://d2znqt9b1bc64u.cloudfront.net/amazon-corretto-${pkg_version}-linux-x64.tar.gz"
+pkg_source="https://d3pxv6yz143wms.cloudfront.net/${pkg_version}/amazon-corretto-${pkg_version}-linux-x64.tar.gz"
```

Comment:
Is there not a non-cloudfront URL we can use?  Perhaps one of these?

https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html

If not, I think a comment on why we use this cloudfront URL directly might be good.